ldk/javascript/src/webpack/generate-banner.ts

```diff
@@ -19,6 +19,7 @@ export function generateMetadata(ldkSettings: LdkSettings): string {
       keyboard: ldkSettings.ldk.permissions.keyboard || undefined,
       network: ldkSettings.ldk.permissions.network || undefined,
       process: ldkSettings.ldk.permissions.process || undefined,
+      system: ldkSettings.ldk.permissions.system || undefined,
```

Comment:
We will most likely have to coordinate with Loop Lib to ensure their permissions parsing can handle this new aptitude.

Reply:
I now have a PR in progress to update the Loop Lib schemas, so I'm going to mark this resolved.